### PR TITLE
Ark flow rev 3 with PAA3905

### PR DESCRIPTION
**Pull request by AlexKlimaj**
With prototype longer range AFBR - https://review.px4.io/plot_app?log=f72b8dd4-da24-43b3-aa53-512300f3db31

Optical flow fusion only. No GPS fusion. - https://review.px4.io/plot_app?log=c2f06366-5749-40ee-a67b-47e08e43ec5e

### CODE REVIEW

File: src/drivers/optical_flow/paa3905/PAA3905.cpp [hunk 1]
```diff
@@ -415,19 +415,14 @@ uint8_t PAA3905::RegisterRead(uint8_t reg)
 		px4_udelay(TIME_TSWR_us - elapsed_last_write);
 	}
 
-	// tSRW/tSRR SPI Time Between Read And Subsequent Commands
-	const hrt_abstime elapsed_last_read = hrt_elapsed_time(&_last_read_time);
-
-	if (elapsed_last_write < TIME_TSRW_TSRR_us) {
```
Comment (mhkabir) [inline src/drivers/optical_flow/paa3905/PAA3905.cpp:421]: This was just wrong I guess? Should have been `elapsed_last_read`?

If that is fixed, is the change still necessary?

Comment (dagar) [reply src/drivers/optical_flow/paa3905/PAA3905.cpp:421]: If we go by the letter of the datasheet we should be respecting these times, but they're lower on newer models and we can get away with it.

**Comment by dagar** [comment]
Fix the astyle fail and we'll bring this in?

**Comment by AlexKlimaj** [comment]
> Fix the astyle fail and we'll bring this in?

Fixed and forced pushed.

**Review by dagar — APPROVED** [review]

**Pull request merged**